### PR TITLE
Feature/enum

### PR DESCRIPTION
**Pull request by nilsvennemann**
I was thinking why not add enum types. These are often used and have an effective solution.

### CODE REVIEW

File: src/Commands/MakeEnumCommand.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class MakeEnumCommand extends Command
+{
+    protected $signature = 'beyond:make:enum {name}';
+
+    protected $description = 'Make a new enum type';
+
+    public function handle()
+    {
+        try {
+            $name = $this->argument('name');
```
Comment (alexgaal) [inline src/Commands/MakeEnumCommand.php:17]: @regnerisch maybe add an option `backed` which automatically creates a backed enum?

Comment (regnerisch) [reply src/Commands/MakeEnumCommand.php:17]: @alexgaal what should be the difference? Will the enum.stub not be the same?

**Comment by regnerisch** [comment]
Documentation ist missing.

File: src/Commands/MakeEnumCommand.php (hunk 1 above)
Comment (alexgaal) [reply src/Commands/MakeEnumCommand.php:17]: It actually would be. Let's not implement it then.

**Review by alexgaal — APPROVED** [review]

**Pull request merged**